src/objects.rs

```diff
@@ -115,6 +147,11 @@ pub enum OsmObj {
     /// A relation
     Relation(Relation),
 }
+impl<'a> AsRef<OsmObj> for &'a OsmObj {
```

Comment:
You don't want that, you want `Borrow`

Reply:
I'm not even sure it's required anymore actually... Let me confirm.

Reply:
Not needed anymore, removed.